### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by mattgle**
## Summary

Update readme

## Screenshots

image

## Notion Ticket
https://www.notion.so/860acccea5a8443aa9479260b8a453fb?v=4f9fe61232824265ba5adebf9dadf5f2&p=4f54ffca969049dd98ec4673f2e686be

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,25 +54,72 @@ The code is built using React-Native and running code locally requires a Mac or
...
+  cd plug-mobile
+  yarn install
+  cd ios && pod install && cd .. # install pods for iOS
+```
+
 - Choose the toolchain and install rust dependencies following the nexts steps:
 
   ```bash
   $ cargo install cargo-lipo
-  $ rustup default 1.55.0-aarch64-apple-darwin
```
Comment (ignaciosantise) [inline README.md:72]: im using this one in M1

File: README.md [hunk 2]
```diff
@@ -54,25 +54,72 @@ The code is built using React-Native and running code locally requires a Mac or
...
+  $ rustup default 1.55.0-x86_64-apple-darwin
   $ rustup target add aarch64-apple-ios x86_64-apple-ios
+  $ rustup target add aarch64-linux-android
+  $ rustup target add armv7-linux-androideabi
+  $ rustup target add i686-linux-android
   ```
 
+- [Download NDK from Android Studio](https://developer.android.com/studio/projects/install-ndk), if you have M1 download the version `24.0.8215888`.
```
Comment (ignaciosantise) [inline README.md:84]: Im using `22.1.7171670` in M1

File: README.md [hunk 3]
```diff
@@ -54,25 +54,72 @@ The code is built using React-Native and running code locally requires a Mac or
...
+- Install android-ndk:
+
+```bash
+brew install android-ndk
```
Comment (ignaciosantise) [inline README.md:88]: i think we dont need this if we download it from Android Studio

File: README.md [hunk 4]
```diff
@@ -54,25 +54,72 @@ The code is built using React-Native and running code locally requires a Mac or
...
+```
+
+- Set in `local.properties` the NDK path like this:
+  `ndk.dir=/Users/your-user-name/Library/Android/sdk/ndk-bundle` or like this `ndk.dir=/Users/your-user-name/Library/Android/sdk/ndk/24.0.8215888`.
+- Set `$NDK_HOME` in enviroment:
+
+```bash
+export NDK_HOME="/usr/local/share/android-ndk"
```
Comment (ignaciosantise) [inline README.md:96]: here im also using `export NDK_HOME="/Users/your-user-name/Library/Android/sdk/ndk/22.1.7171670"`

File: README.md (hunk 3 above)
Comment (mattgle) [reply README.md:88]: This is needed, we tried it first without this and didnt work

Comment (ignaciosantise) [reply README.md:88]: But did you try changing also the NDK_HOME?
i deleted android-ndk from my computer

File: README.md [hunk 5]
```diff
@@ -54,24 +54,89 @@ The code is built using React-Native and running code locally requires a Mac or
...
+And then run:
+
+```bash
+$ rustup target add aarch64-apple-ios x86_64-apple-ios
+$ rustup target add aarch64-linux-android
+$ rustup target add armv7-linux-androideabi
+$ rustup target add i686-linux-android
+```
+
+- [Download NDK from Android Studio](https://developer.android.com/studio/projects/install-ndk), if you have M1 download the version `22.1.7171670`.
```
Comment (ignaciosantise) [inline README.md:107]:
```suggestion
- [Download NDK from Android Studio](https://developer.android.com/studio/projects/install-ndk) -> Download the version `22.1.7171670`.
```

**Review by ignaciosantise — APPROVED** [review]

**Pull request merged**